packages/liveblocks-emails/src/lib/sanitize-inline-css.ts

```diff
@@ -0,0 +1,147 @@
+const ALLOWED_PROPERTIES = [
```

Comment:
Especially after [this change](https://github.com/liveblocks/liveblocks/pull/1919#discussion_r1791924738), yeah I think it could lead to more issues (an endless cat-and-mouse game with us catching up with new properties and email client changes) than benefits

Reply:
Agree 👍🏻 I will get rid of this 🙂